### ISSUE TITLE
Google icon alt text

### ISSUE THREAD

**Issue by anormand** [unimelb/The-University-of-Melbourne-Web-Templates#7]
### Problem:

The Google icon in the translate pop-up is missing ALT text.
### WCAG Guideline

Success Criteria 1.1.1 – Non-text Content

All images, form image buttons and image map hot spots have equivalent alternative text. Images that do not convey content, are decorative or with content that is already conveyed in text are given null alt text (alt=””) or implemented as CSS backgrounds. Form inputs have associated labels.
### Remedy:

Add ALT text to the image.
### Example

Change
`<img src="http://translate.googleapis.com/translate_static/img/mini_google.png" style="position: relative; top: 5px;">`

to
`<img alt="google" src="http://translate.googleapis.com/translate_static/img/mini_google.png" style="position: relative; top: 5px;">`

**Comment by ptagell** [comment]
Interesting - I thought we'd fixed this. Will investigate.

**Comment by anormand** [comment]
This is an old one from my report audit-report-marcom-templates-0-9-2.docx that we went through together last week.

The are a number of other things that I thought we changed which aren't in 0.9.3 including the repositioning of the More button and the label for 'Translate this page'.

**Comment by kolber** [comment]
These issues were lost somewhere in our commit history. They have been restored and will be released this afternoon.

**Comment by anormand** [comment]
Still not fixed in 0.9.4

**Comment by kolber** [comment]
This sits separately to the templates (within injection), so is being deployed this afternoon.

**Comment by kolber** [comment]
This has just been deployed.
